src/app/study-flashcard/study-flashcard.component.ts

```diff
@@ -1,19 +1,91 @@
-import { Component, HostListener } from '@angular/core';
+import { Component, HostListener, ViewChild } from '@angular/core';
 import { FlashcardComponent } from '../flashcard/flashcard.component';
 import { ReturnRibbonComponent } from '../return-ribbon/return-ribbon.component';
+import { StudySetService } from '../services/study-set.service';
+import { FlashcardData } from '../data-models/flashcard-model';
+import { StudySetData } from '../data-models/studyset-model';
+import { CommonModule } from '@angular/common';
 
 @Component({
   selector: 'app-study-flashcard',
   standalone: true,
-  imports: [FlashcardComponent, ReturnRibbonComponent],
+  imports: [FlashcardComponent, ReturnRibbonComponent, CommonModule],
   templateUrl: './study-flashcard.component.html',
   styleUrl: './study-flashcard.component.scss'
 })
 export class StudyFlashcardComponent {
-  cardScaleFactor: number = window.innerWidth * (880 / 1280) / 500;
+  constructor(private studySetService: StudySetService) { }
+
+  cardScaleFactor: number = window.innerHeight * (496 / 720) / 282;
+  //property named 'flashcards':
+  flashcards: FlashcardData[] = []; 
+
+  //This is the current flashcard we are on. 
+  //currentFlashcard is the name and FlashcardData is the data type. The () means it is a default...
+  //contstructor that is being called. We are creating a new flashcard with keyword new. 
+  currentFlashcard: FlashcardData = new FlashcardData();
+  /*
+  'flashcards' property is an array that holds objects of type FlashcardData.
+  The part after the equal sign is an empty array. 
+  We are assigning the array of type FlashCard Data to an empty array.
+  */
+  currentCardIndex: number = 0;
```

Comment:
remove comments. It is obvious from the code what is happening and the comment make the code harder to read. The comment above` currentCardIndex` isn't even in the right place.

Reply:
I fixed this